frontend/src/scenes/experimentation/FeatureFlag.tsx

```diff
@@ -0,0 +1,356 @@
+                                />
+                            </Form.Item>
+
+                            <Collapse>
+                                <Collapse.Panel
+                                    header={
+                                        <div style={{ display: 'flex', fontWeight: 'bold' }}>
+                                            <img src={jsLogo} height={16} style={{ marginRight: 6 }} /> Javascript
+                                            integration instructions
+                                        </div>
+                                    }
+                                    key="js"
+                                >
+                                    <Form.Item
+                                        shouldUpdate={(prevValues, currentValues) =>
+                                            prevValues.key !== currentValues.key
+                                        }
+                                    >
+                                        {({ getFieldValue }) => <JSSnippet flagKey={getFieldValue('key')} />}
+                                    </Form.Item>
+                                </Collapse.Panel>
+                                <Collapse.Panel
+                                    header={
+                                        <div style={{ display: 'flex', fontWeight: 'bold' }}>
+                                            <img src={pythonLogo} height={16} style={{ marginRight: 6 }} /> Python
+                                            integration instructions
+                                        </div>
+                                    }
+                                    key="python"
+                                >
+                                    <Form.Item
+                                        shouldUpdate={(prevValues, currentValues) =>
+                                            prevValues.key !== currentValues.key
+                                        }
+                                    >
+                                        {({ getFieldValue }) => <PythonSnippet flagKey={getFieldValue('key')} />}
+                                    </Form.Item>
+                                </Collapse.Panel>
+                            </Collapse>
+                        </Col>
+                        <Col span={24} md={12}>
+                            <h3 className="l3">Release condition groups ({featureFlag.filters.groups.length})</h3>
+                            <div className="text-muted mb">
+                                Specify which users or groups of users to which you want to release this flag.
+                            </div>
+                            {featureFlag.filters.groups.map((group, index) => (
+                                <Card
+                                    style={{ position: 'relative', marginBottom: 32, paddingBottom: 48 }}
```

Comment:
Feels like there is a bit too much margin in this entire card, both top+bottom and between add filter and rollout. It's nice to be able to see a couple of groups without having to scroll.
![image](https://user-images.githubusercontent.com/1727427/111620450-07069880-87e7-11eb-84b3-f3c830a79e69.png)

Reply:
I've reduced the bottom padding, everything else is our standard 16px, I'm a bit hesitant to reduce it for consistency, but let me know what you think.